classes/Renderer.php

```diff
@@ -443,6 +446,33 @@ public static function output($template, $data = array())
         require_once INCLUDESPATH . 'easyparliament/templates/html/footer.php';
     }
 
+    private static function get_banner_text() {
+        $text = NULL;
+
+        global $memcache;
```

Comment:
As this appears a few times here (and elsewhere), perhaps worth a small class to replace this global, that performs the connection when necessary and stores if needed?